src/libraries/System.Formats.Cbor/tests/System.Formats.Cbor.Tests.csproj

```diff
@@ -32,10 +30,25 @@
     <Compile Include="Writer\CborWriterTests.TextString.cs" />
     <Compile Include="CoseKeyHelpers.cs" />
   </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'">
+    <Compile Include="CborTestHelpers.netcoreapp.cs" />
+    <Compile Include="PropertyTests\CborPropertyTests.cs" />
+    <Compile Include="PropertyTests\CborRandomGenerators.cs" />
+    <Compile Include="Reader\CborReaderTests.Simple.netcoreapp.cs" />
+    <Compile Include="Writer\CborWriterTests.Simple.netcoreapp.cs" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' != '$(NetCoreAppCurrent)'">
+    <Compile Include="CborTestHelpers.netstandard.cs" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'">
+    <ProjectReference Include="CborDocument\System.Formats.Cbor.Tests.DataModel.fsproj" />
```

Comment:
It might be worth in the future to attempt porting the random tests to net48 as well, since it might let us catch any bugs not covered by the unit test suite. I can help out with the F# components if you want.